### ISSUE TITLE
Unable to build the latest release from AUR

### ISSUE THREAD

**Issue by Azarattum** [houmain/keymapper#25]
Build fails with:
```
fatal: No names found, cannot describe anything.
CMake Error at CMakeLists.txt:153 (list):
list GET given empty list

CMake Error at CMakeLists.txt:154 (list):
list GET given empty list

CMake Error at CMakeLists.txt:155 (list):
list GET given empty list

— Configuring incomplete, errors occurred!
```

Commenting out these 3 lines makes it compile, but the version info seem to be broken.
```
...
#list (GET VERSION_LIST 0 CPACK PACKAGE VERSION_MAJOR)
#list (GET VERSION_LIST 1 CPACK PACKAGE VERSION_MINOR)
#list (GET VERSION_LIST 2 CPACK PACKAGE VERSION_PATCH)
...
```
```
$ ./keymapper —help
keymapper  (c) 2019-2022 by Albert Kalchmair
...
```

**Comment by Azarattum** [comment]
Also `src/_version.h` contains just empty quotation marks.
```
""
```

**Comment by Azarattum** [comment]
Hmm... Interesting. The cloned from git version compiles just fine. It creates a valid `_version.h` file at the `cmake -B build` step.

**Comment by Azarattum** [comment]
`keymapper-git` AUR package works perfectly fine.

**Comment by houmain** [comment]
Thanks for the thorough report. The last commit should fix the problem (d11ad2ab46eb1d30804360499e5c95e42de807c6).
Now it builds even when it could not retrieve the version.
The version can also be passed to CMake using e.g. -DVERSION=1.8.3
